roles/openshift_logging_elasticsearch/tasks/main.yaml

```diff
@@ -207,7 +207,7 @@
       name: openshift_logging
       tasks_from: patch_configmap_files.yaml
     vars:
-      configmap_name: "logging-elasticsearch"
+      configmap_name: "{{ elasticsearch_name }}"
```

Comment:
var that is passed in that allows us to have separate configmaps between ops and non-ops?

Reply:
we currently do that when creating. The problem was we were only patching based on the non-ops configmap so we would blow away any changes that were specific to only ops configmaps